Extensions/old_notifications.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE Classic Notifications **//
-//* VERSION 0.6 REV B **//
+//* VERSION 0.6.1 **//
```

Comment:
Version number was incremented incorrectly. `REV B` + 1 would be `REV C`, which maps to `0.6.3`.